proto/api.proto

```diff
@@ -472,6 +472,9 @@ message PushTxnResponse {
 // commit them.
 message ResolveIntentRequest {
   optional RequestHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
+  // The successfully pushed intent txn. Success means either the txn
```

Comment:
updated the comment: 
>   // The transaction whose intent is being pushed. The transaction must either
  // be COMMITTED or ABORTED.

intent_txn being nullable makes no sense - it's required. You can't resolve an intent without a `Txn`.